ci/gpu/build.sh

```diff
@@ -66,8 +66,8 @@ fi
 
 conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
               "dask>=2.15.0" "distributed>=2.15.0" "numpy>=1.16" "double-conversion" \
-              "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.6.0" "dlpack" \
-              "feather-format" "cupy>=6.6.0,<8.0.0a0,!=7.1.0" "arrow-cpp=0.15.0" "pyarrow=0.15.0" \
+              "rapidjson" "flatbuffers" "boost-cpp=1.72.0" "fsspec>=0.6.0" "dlpack" \
```

Comment:
Awesome! 🎉 Someone was asking about this earlier 😄

Reply:
That was the reason for the conflict in solving the environment 😄

Reply:
Should we move this into the recipe then?

Reply:
Yea into the integration recipe, but we shouldn't need to pin within libcudf as the only thing we use is `boost::filesystem` which typically doesn't have ABI breaking changes.

Reply:
Sure though we can make sure we are building with at least `1.72.0`.